### ISSUE TITLE
Screen freezing and flickering

### ISSUE THREAD

**Issue by fh2150** [citra-emu/citra#5587]
Issue:
I just got UltraSun for Citra and O have the latest version of it and I am using the nightly mingw folder, and whenever I start it up, it plays well, but then as I start gameplay, little black boxes constantly pop up on the emulator. I tried resetting back to default settings, updated my and tried othert things and it still happens. Any help will be appreciated, but keep in layman terms, please.

Windows 10 Home: 64-bit processor
CPU:Intel Core i3-1005G1 CPU 1.20 GHz 1.19 GHz
GPU: Intel UHD Graphics
Citra Version (found in title bar): Nightly 1618
Game: Pokemon Ultra Sun
Screenshot of Issue (include the full Citra window including titlebar):

![image](https://user-images.githubusercontent.com/73228439/97116351-d5275500-16d2-11eb-8dbf-732d00207066.png)

**Comment by fh2150** [comment]
https://community.citra-emu.org/uploads/short-url/irfUgTlHGdtpmZW4dPKs8Wh7Hq1.txt

**Comment by fh2150** [comment]
For further reference...

https://community.citra-emu.org/t/little-black-boxes/300756/76

**Comment by fh2150** [comment]
https://drive.google.com/file/d/15vsBfTEX5ywVYwxIkg0GUIwluNHxkwd4/view

**Comment by fh2150** [comment]
https://discord.com/channels/@me/768429728867942400

**Comment by fh2150** [comment]
![image](https://user-images.githubusercontent.com/73228439/97117003-757f7880-16d7-11eb-92a5-b12d399e111f.png)

**Comment by tywald** [comment]
Remember that this is an issue tracker for developers and not a support forum so you have to formulate your report appropriately.

Graphics driver version is absent, you should include it. If it is not up to date you should update it first.

The "screen freeze problem" has a crash related to it (information from Discord). Here is an exception thrown with a call stack:

**Comment by fh2150** [comment]
Ohh ok. 

intel UHD Graphics version 26.20.100.8141

**Comment by fh2150** [comment]
[citra_log.txt](https://github.com/citra-emu/citra/files/5438947/citra_log.txt)


Now the screen is completely black, not showing anything, but the game is still functioning......

**Comment by fh2150** [comment]
This happened after updating the driver.....

**Comment by Schplee** [comment]
This is an issue tracker, not a support forum. User's issue was resolved on our forum. Closing

**Comment by fh2150** [comment]
Thank you for closing it. Here is the link to the solution

https://www.youtube.com/watch?v=rPuvOpaCmyY&ab_channel=STS
